packages/elements-dev-portal/src/handlers/getNodes.ts

```diff
@@ -2,31 +2,41 @@ import { NodeSearchResult } from '../types';
 import { appVersion } from '../version';
 
 export const getNodes = async ({
-  workspaceId,
   projectIds,
+  workspaceId,
   search,
   platformUrl = 'https://stoplight.io',
   platformAuthToken,
 }: {
-  workspaceId: string;
+  projectIds: string[];
+  workspaceId?: string;
   search?: string;
-  projectIds?: string[];
   platformUrl?: string;
   platformAuthToken?: string;
 }): Promise<NodeSearchResult[]> => {
   const queryParams = [];
+  let fetchedWorkspaceId = workspaceId || '';
 
-  if (projectIds && projectIds.length) {
-    queryParams.push(...projectIds.map((projectId, index) => `project_ids[${index}]=${projectId}`));
+  if (!workspaceId) {
+    const response = await fetch(`${platformUrl}/api/v1/projects/${projectIds[0]}`, {
```

Comment:
Hmm, I am on the fence here.

On one hand it will likely work quite well, and it is a fast fix.

On the other, the cleaner solution would be to actually update elements-service, to not require workspaceId when searching nodes... 🤔 

@wmhilton would appreciate your opinion here.

Reply:
Yeah, agree! I borrowed this from `getNodeContent` because it was quicker/easier for me to handle here. But, I am open to approaching from the service side. If that's preferred, I may need some additional help getting acquainted over there.

Reply:
I kind of think that editing the service should be a different issue because the scope is getting quite large. Feel free to push back on me!

Reply:
Hmmmmmm. I agree with @mpodlasin that it would be cleaner to just fix up the API so we didn't need the workspace id. And while `/v1/projects/{project_id}/nodes` would be an obvious choice (seeing how the other paths are named)

![image](https://user-images.githubusercontent.com/587740/133155423-143b45a8-cbcc-4536-81f6-9dd7d8771682.png)

That would mean changing the hook so you can only pass one `project_id`? And IDK how to modify the endpoint to handle multiple ids. Would it be `/v1/projects/{comma_separated_ids}/nodes`? What other complicated things would turn up?

So... I'm inclined to just shrug and say "sure" for now? Weird to have to make two HTTP requests but whatever. Let's figure that out in a new issue I guess.